### PR TITLE
Add Notification Detail Page

### PR DESCRIPTION
**Pull request by dhaselhan**
* Timeline, Overview, Staff Journal, App Info, App Preview, etc.
* Add missing Transferees section to the portal

### CODE REVIEW

**Review by mhuseinov — APPROVED** [review]

File: alcs-frontend/src/app/shared/time-tracker/time-tracker.component.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class TimeTrackerComponent implements OnChanges {
   dueDate: Date | null = null;
   isOverdue = false;
 
-  @Input() data: TimeTrackable | undefined;
+  @Input() data: TimeTrackable | any | undefined;
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply alcs-frontend/src/app/shared/time-tracker/time-tracker.component.ts]: Fixed! Realized I could use another variable.

File: alcs-frontend/src/app/shared/header/search-bar/search-bar.component.ts [hunk 2]
```diff
@@ -60,6 +60,10 @@ export class SearchBarComponent {
           case 'NOI':
             await this.router.navigate(['notice-of-intent', result.referenceId]);
             break;
+          case 'NOTI':
+            await this.router.navigate(['notification', result.referenceId]);
+            debugger;
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply alcs-frontend/src/app/shared/header/search-bar/search-bar.component.ts]: Removed.

File: alcs-frontend/src/app/shared/details-header/details-header.component.html [hunk 3]
```diff
@@ -47,9 +47,9 @@ <h5>{{ _application.fileNumber }} ({{ _application.applicant }})</h5>
       </div>
     </div>
     <div>
-      <ng-container *ngIf="!_application.hasOwnProperty('decisionDate')">
+      <ng-container *ngIf="isTimeTrackable && !_application.hasOwnProperty('decisionDate')">
```
Comment (mhuseinov) [inline alcs-frontend/src/app/shared/details-header/details-header.component.html:50]: I'm confused, but it looks like the isTimeTrackable is always false, correct?

Comment (dhaselhan) [reply alcs-frontend/src/app/shared/details-header/details-header.component.html:50]: Removed isTimeTrackable

**Pull request merged**